src/server/test_utils.h

```diff
@@ -46,6 +54,13 @@ class BaseFamilyTest : public ::testing::Test {
     return Run(ArgSlice{list.begin(), list.size()});
   }
 
+  RespExpr RunAdmin(std::initializer_list<const std::string_view> list) {
```

Comment:
I'd add a small doc line, something along the lines of "Runs the command in a mocked admin connection"